user_guide_src/source/changelogs/v4.4.0.rst

```diff
@@ -70,6 +72,8 @@ Message Changes
 Changes
 *******
 
+- The ``DownloadResponse`` class, when generating response headers, does not replace the ``Content-Disposition`` header if it was previously specified.
+
```

Comment:
```diff
--- a/user_guide_src/source/changelogs/v4.4.0.rst
+++ b/user_guide_src/source/changelogs/v4.4.0.rst
@@ -29,8 +29,6 @@ Method Signature Changes
 Enhancements
 ************
 
-- Added ``DownloadResponse::inline()`` method that sets the ``Content-Disposition: inline`` header to display the file in the browser.
-
 Commands
 ========
 
@@ -61,6 +59,9 @@ Helpers and Functions
 Others
 ======
 
+- **DownloadResponse:** Added ``DownloadResponse::inline()`` method that sets
+  the ``Content-Disposition: inline`` header to display the file in the browser.
+  See :ref:`open-file-in-brower` for details.
 - **View:** Added optional 2nd parameter ``$saveData`` on ``renderSection()`` to prevent from auto cleans the data after displaying. See :ref:`View Layouts <creating-a-layout>` for details.
 - **Auto Routing (Improved)**: Now you can use URI without a method name like
   ``product/15`` where ``15`` is an arbitrary number.
```

Reply:
Sorry, I pasted it in the wrong place.
My intention is to move the Enhancement item to the section `Enhancements > Others`.